types.ts

```diff
@@ -1342,6 +1347,32 @@ export interface AssistantMessageTool extends BaseTool<ToolType.AssistantMessage
  */
 export interface SummarizerTool extends BaseTool<ToolType.Summarizer> {}
 
+/**
+ * Tool that provides access to MCP capabilities.
+ * @hidden
+ */
+export interface MCPTool extends BaseTool<ToolType.MCP> {
+  /**
+   * The names of the MCP servers added to this pack that this tool can connect to.
+   */
+  serverNames?: string[];
```

Comment:
Shouldn't this be required?

Reply:
@ekoleda-codaio had a doc where unspecified server names meant all servers in the pack.